Makefile.am

```diff
@@ -76,3 +69,8 @@ TAG.txt:
 		TAG="NO_GIT"; \
 	fi ; \
 	echo "$$TAG" > TAG.txt
+
+W=@OVIS_INCLUDE_REL@
+X=@OVIS_INCLUDE_ABS@
+Y=@OVIS_LIB_REL@
+Z=@OVIS_LIB_ABS@
```

Comment:
Were these stray testing lines that need to be removed now?

Reply:
Thanks for the catches. I repushed with all the suggested fixes, except helper-scripts which has always been an orphan; documented that.